site/app/views/OfficeHoursQueueView.php

```diff
@@ -62,6 +63,17 @@ public function showQueueStudentStats($studentData) {
         ]);
     }
 
+    public function previewAnnouncement($enablePreview, $content) {
+        $this->core->getOutput()->disableRender();
+        if (!$enablePreview) {
+            return;
+        }
+        return $this->core->getOutput()->renderTwigTemplate("misc/Markdown.twig", [
+                "style" => "font-family: 'Source Sans Pro', sans-serif;",
```

Comment:
A general question, why do some views have this style applied and some do not? Could we unify all places to have the same font for displaying markdown, which would be preferable to maintain consistency throughout the site.

Reply:
Good point, initially these places were missing the font that the others used so I temporarily just inserted it there. I just pushed a change to move the styling to the markdown class that way it is always used in all places.